Corona-Warn-App/src/main/res/values/srs_submission_strings.xml

```diff
@@ -11,4 +11,49 @@
     <!-- XTXT: Register test card button -->
     <string name="register_test_card_button">Weiter</string>
 
-</resources>
\ No newline at end of file
+    <!-- Submission Consent Screen -->
+    <!-- XHED: Submission consent header -->
+    <string name="srs_submission_consent_header">"Ihr Einverständnis"</string>
```

Comment:
Please add these resources to DE file , VERY important otherwise it won't get translated , any legal text should got to the legal strings files, DE, EN, TR, Legal text is usually highlighted with GREY box, Better to double check it with Martin

Reply:
Added strings from grey box to legal string file, which doesn't apply to srs_submission_consent_header exactly because it is used as the navigation title and is not used in grey box